apps/api/src/app/content-templates/usecases/compile-template/compile-template.usecase.ts

```diff
@@ -7,7 +7,7 @@ import { CompileTemplateCommand } from './compile-template.command';
 
 Handlebars.registerHelper(HandlebarHelpersEnum.EQUALS, function (arg1, arg2, options) {
   // eslint-disable-next-line eqeqeq
-  return arg1 == arg2 ? options.fn(this) : options.inverse(this);
+  return arg1 === arg2 ? options.fn(this) : options.inverse(this);
```

Comment:
So this one was actually intended :) Could you revert it back please? her it's checking `1 == '1'` so type conversion is expected